css-index.css

```diff
@@ -0,0 +1,76 @@
+@media (min-width:900px){
+  .container{
+    width: auto;
+    border: 3px solid black;
+    display: grid;
+    grid-template-columns: repeat(4,1fr);
+    grid-auto-rows: 100px;
+  }
+  .container > div{
+    border: 1px solid black;
+  }
+  .navegation{
+    display: flex;
+    align-self: flex-end;
+    grid-column: 1/5;
+  }
+  .name{
+    display: flex;
```

Comment:
this is only a div, why should have the property flex? there is not anything inside 🤔